pkgs/development/python-modules/opensimplex/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, autopep8
+, coverage
+, nose
+, pycodestyle
+, twine
+}:
+
+buildPythonPackage rec {
+  pname = "opensimplex";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "lmas";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "idF5JQGnAye6z3c3YU9rsHaebB3rlHJfA8vSpjDnFeM=";
+  };
+
+  checkInputs = [ autopep8 coverage nose pycodestyle twine ];
+  checkPhase = "make test";
```

```suggestion
  checkPhase = ''
    nosetests tests/
  '';
```
we do not want to generate converage.